### PR TITLE
[Themes] - Use shimmer for loading the carousel

### LINKED ISSUE #10414: Show shimmering/skeleton view for theme screenshots while they are loading
Avoid blank screenshot items when the Theme preview screenshot is being loaded:

 

Add a shimmering effect to the empty view while loading.

### PR DESCRIPTION
**Pull request by 0nko**
Fixes #10414. The PR adds shimmer for loading the theme picker carousel.

**To test:**
1. Go To More menu -> Settings -> Themes
2. Notice a shimmer effect is used until the theme preview screenshot are loaded

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerScreen.kt [hunk 1]
```diff
@@ -137,7 +138,7 @@ private fun CurrentTheme(
             is CurrentThemeState.Loading -> {
                 SkeletonView(
                     width = dimensionResource(id = R.dimen.skeleton_text_medium_width),
-                    height = dimensionResource(id = R.dimen.skeleton_text_height_100)
+                    height = 22.dp
```
Comment (hichamboushaba) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerScreen.kt:141]: A question: Why did you change this value @0nko? I used it to keep a consistent value with the other usages of skeleton in the app.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerScreen.kt [hunk 2]
```diff
@@ -212,17 +213,30 @@ private fun ColumnScope.Carousel(
...
+            .horizontalScroll(rememberScrollState()),
+        horizontalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.major_100))
     ) {
-        CircularProgressIndicator(
-            modifier = Modifier
-                .align(Alignment.Center)
-        )
+        repeat(6) {
+            val startPadding = if (it == 0) dimensionResource(id = R.dimen.major_100) else 0.dp
+            val endPadding = if (it == 5) dimensionResource(id = R.dimen.major_100) else 0.dp
```
Comment (hichamboushaba) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerScreen.kt:225]: Just wondering, won't applying an horizontal padding to the `Row` itself achieve the same effect as this?

**Review by hichamboushaba — APPROVED** [review]
Thanks @0nko for working on this, it works well, I left two minor questions, but they are not blocking.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerScreen.kt (hunk 1 above)
Comment (0nko) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerScreen.kt:141]: The SkeletonView's height was different than the final text, which would cause a jump.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerScreen.kt (hunk 2 above)
Comment (0nko) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerScreen.kt:225]: No, because that would add a padding to the right edge, while we want to make the carousel item go up the edge (out of viewport).

Comment (hichamboushaba) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerScreen.kt:225]: Oh, you are right, thanks for the clarification.
